### PR TITLE
Additional sorting capability

### PR DESCRIPTION
**Pull request by nathancashmoredwp**
Includes change from alpha to title and deadline data and status search.  More tests required or not ?

### CODE REVIEW

File: app/controllers/jobs-controller.js [hunk 1]
```diff
@@ -37,7 +37,8 @@ router.post('/new', (req, res, next) => {
       new AddJobViewModel(accountId, req.body, req.validationErrors()));
   }
 
-  const jobData = Object.assign({}, req.body, { accountId, status: progression[0] },
+  const jobData = Object.assign({}, req.body,
+    { accountId, status: progression[0], status_sort_index: 0 },
     { deadline: parseDeadline(deadline) });
```
Comment (gjvis) [inline app/controllers/jobs-controller.js]: Might be simpler to have only one object that gets merged with the request body?

```javascript
const jobData = Object.assign({}, req.body, {
  accountId,
  deadline: parseDeadline(deadline),
  status: progression[0],
  status_sort_index: 0,
});
```

File: test/common/page_objects/add-job-page.js [hunk 2]
```diff
@@ -14,6 +14,7 @@ const AddJobPage = function AddJobPage(browser) {
   this.getValidationError = () => browser.text('#validation-errors');
 
   this.visit = (accountId) => browser.visit(`/${accountId}/jobs/new`);
+  this.patch = (accountId, jobId) => browser.visit(`/${accountId}/jobs/${jobId}?_method=PATCH`);
```
Comment (gjvis) [inline test/common/page_objects/add-job-page.js]: This method doesn't appear to be used anywhere

Comment (nathancashmoredwp) [inline test/common/page_objects/add-job-page.js]: Yes, i thought you'd notice that ... it fell in line with the "more tests required" line.

Comment (gjvis) [inline test/common/page_objects/add-job-page.js]: Ah of course, sorry I should have realised.

I think its worth adding an integration test for the change to the patch method, its the sort of behaviour that is just non-obvious enough for us to forget about

**Comment by gjvis** [comment]
👍

**Pull request merged**